lib/modules/deployment/contract_deployer.js

```diff
@@ -153,7 +153,13 @@ class ContractDeployer {
 
           self.blockchain.getCode(trackedContract.address, function(_getCodeErr, codeInChain) {
             if (codeInChain !== "0x") {
-              self.contractAlreadyDeployed(contract, trackedContract, next);
+              //  if contract deployed, deploy the contract regardless when track field is defined and set to false
+              if (typeof trackedContract.track === 'boolean' && trackedContract.track === false) {
```

Comment:
Actually, this if should be above ` self.blockchain.getCode` since we want to skip tracking altogether. So basically, if track === false, just redeploy and skip checking.